CHANGELOG.md

```diff
@@ -8,6 +8,55 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.5] - 2023-02-14
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.4...v0.3.5)
+
+### Added
+* [`grpc-health-probe`](https://github.com/grpc-ecosystem/grpc-health-probe) for Health Checks (#520)
+
+  OpenFGA containers now include an embedded `grpc_health_probe` binary that can be used to probe the Health Check endpoints of OpenFGA servers. Take a look at the `./docker-compose.yaml` file for an example.
+
+* Improvements to telemetry: logging, tracing, and metrics (#468, #514, #517, #522)
+
+  * We have added Prometheus as the standard metrics provided for OpenFGA and provide a way to launch Grafana to view the metrics locally. See `./docker-compose.yaml` for more information.
+
+  * We've improved the attributes of various trace spans and made sure that trace span names align with the functions they decorate.
+
+  * Our logging has been enhanced with more logged fields including request level logging which includes a `request_id` and `store_id` field in the log message.
+
+  These features will allow operators of OpenFGA to improve their monitoring and observability processes.
+
+* Nightly releases (#508) - thanks @Siddhant-K-code!
+
+  You should now be able to run nightly releases of OpenFGA using `docker pull openfga/openfga:nightly`
+
+### Fixed
+* Undefined computed relations on tuplesets now behave properly (#532)
+
+  If you had a model involing two different computed relations on the same tupleset, then it's possible you may have received an internal server error if one of the computed relations was undefined. For example,
+  ```
+  type document
+    relations
+      define parent as self
+      define viewer as x from parent or y from parent
+
+  type folder
+    relations
+      define x as self
+
+  type org
+    relations
+      define y as self
+  ```
+  Given the tuple `document:1#parent@organization:x`, then `Check(document:1#viewer@jon)` would return an error prior to this fix because the `x` computed relation on the `document#parent` tupleset relation is not defined for the `org` object type.
+
+* Eliminate duplicate objects in ListObjects response (#528)
+
+* Error handling for Check when a provided authorization model id is not found (#541)
+
+  The `Check` API would return an internal server error if you provided an authorization model id in the request that didn't reference an existing model. This fixes that so you get a request validation error instead.
+
```

Comment:
Update with https://github.com/openfga/openfga/pull/544?

Reply:
I guess #544 and #541 technically don't need to be included, because they aren't part of a prior release yet anyways..

Reply:
I've removed the bits that are for code that fixes stuff that is not yet releases 😄